packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -39,6 +39,10 @@ export function errorHandling(errorDetails: any, label?: string, moreInfo?: stri
         "Invalid Credentials. Please ensure the username and password for {0} are valid or this may lead to a lock-out.",
         label
     );
+    const errToken = localize(
+        "errorHandling.invalid.token",
+        "Your connection is no longer active. Please log in to the API Mediation Layer to restore the connection."
```

Comment:
this error could be more generalized to include alternate profile types whose token has expired, "Please log in to restore the connection."